lilac2/api.py

```diff
@@ -58,8 +58,9 @@ def add_into_array(which: str, extra_deps: Iterable[str]) -> None:
   '''
   field_appeared = False
 
+  pattern = re.compile(which + r'=')
   for line in edit_file('PKGBUILD'):
-    if line.strip().startswith(which):
+    if pattern.match(line):
```

Comment:
What if there is indentation?

Reply:
Good question...
Then... what if that's another variable defined in a function?

Reply:
In that senario maybe not use this at all?